frontend/app/src/components/asset-manager/MergeDialog.vue

```diff
@@ -113,14 +113,14 @@ const v$ = useVuelidate(
       <div v-if="done">{{ t('merge_dialog.done') }}</div>
 
       <v-form v-else :value="!v$.$invalid">
-        <asset-select
+        <v-text-field
```

Comment:
When we did the change I was pretty sure that we had a reason for this 😄 
Thanks for confirming. 

Can you add a comment above it so that we don't accidentally remove it in the future?